src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -172,7 +172,12 @@ export const AddressModal: React.FC<Props> = ({
         >
           {(formik: FormikProps<SavedAddressType>) => (
             <form onSubmit={formik.handleSubmit}>
-              <Text data-test="credit-card-error" color="red" my={2}>
+              <Text
+                variant="xs"
+                data-test="credit-card-error"
+                color="red100"
+                my={2}
+              >
```

Comment:
A little difficult to tell if this is appropriate but we have an `error` variant `Banner` for these use-cases:

https://palette-storybook.artsy.net/?path=/story/components-banner--default